fenix/app/src/main/res/values/strings.xml

```diff
@@ -2235,9 +2235,9 @@
     <!-- Flag showing that the review quality check feature is work in progress. -->
     <string name="review_quality_check_beta_flag" tools:ignore="UnusedResources">Beta</string>
     <!-- Content description (not visible, for screen readers etc.) for opening browser menu button to open review quality check bottom sheet. -->
-    <string name="browser_menu_review_quality_check">Open Review quality check</string>
+    <string name="browser_menu_review_quality_check_1">Open review checker</string>
```

Comment:
Since the string will be the same as `review_quality_check_second_cfr_action`, I am wondering if we should've have this as a separate string. I do understand having separate strings for content description.

Reply:
I think it's good to have them in separate strings, in case the CFR action string changes/gets removed in the future we'll still have the content description unaffected.